docs/guides/options.md

```diff
@@ -28,8 +28,11 @@
   * [language](#language)
   * [languages](#languages)
   * [liveui](#liveui)
+  * [liveTracker.trackingThreshold](#livetrackertrackingthreshold)
+  * [liveTracker.liveTolerance](#livetrackerlivetolerance)
```

Comment:
I don't know where these two lines came from?! I can remove them?

Reply:
If you've run `npm run docs:fix`, it probably added these two items that were added to the doc but we missed updating the TOC with them. https://github.com/videojs/video.js/pull/7097

Reply:
@gkatsev so it would be okay to merge this change and let those two lines get added in `options.md`? Or would you prefer to have them in a separate PR?

Reply:
They can go in here.